extensions/default/src/getHangingProtocolModule.js

```diff
@@ -9,6 +9,15 @@ const defaultProtocol = {
   editableBy: {},
   protocolMatchingRules: [],
   toolGroupIds: ['default'],
+  displaySetSelectors: {
+    displaySet: {
```

Comment:
can we say `deafultDisplaySetId` here so that someone don't confuse it with actual `DisplaySet` here?